lib/dirac_clover_hasenbusch_twist.cpp

```diff
@@ -33,25 +33,25 @@ namespace quda
     if (!asymmetric) {
       if (matpcType == QUDA_MATPC_EVEN_EVEN) {
```

Comment:
Can we formulate this conditional in terms of `this_parity`/`other_parity` or not really?

Reply:
Good suggestion c5b1081ad12a91e07dc510fb97519801b8ecdbe8